pkg/metric_generator/generator_test.go

```diff
@@ -0,0 +1,241 @@
+package generator
+
+import (
+	"k8s.io/kube-state-metrics/pkg/allow"
```

Comment:
Need to apply goimports here. Let's separate the stdlib and third-party imports into different import groups.

Consider using this tool: https://github.com/incu6us/goimports-reviser

Reply:
Done